content/_data/changelogs/weekly.yml

```diff
@@ -915,6 +915,47 @@
     # pull: 2960 not notable enough
     # pull: 2963 not notable enough
     # pull: 2964 not notable enough
+- version: "2.74"
+  date: 2017-08-13
+  changes:
+    - type: major rfe
+      message: >
+        Upgrade Stapler library from 1.250 to 1.252
+      references:
+        - pull: 2956
```

Comment:
Well, I need to update the changelog :)
IMHO JENKINS-45903 really makes sense, because it is an important defect (file leak). And it's not obvious that such logic comes from Stapler.